src/components/tools/Map/VectorLayer.svelte

```diff
@@ -52,10 +62,16 @@
 
   if (enableClick) {
     map.on("click", layer.id, function (e) {
-      dispatch("overlayclick", e.features[0].id);
-      map.flyTo({
-        center: e.features[0].geometry.coordinates,
-      });
+      const feature = e.features && e.feature.length ? e.feature[0] : null;
```

Comment:
@mukhtyar if `feature` is `null` then calling `getFeatureCenter` will throw an error because it checks for `feature.geometry`.